src/utils.js

```diff
@@ -227,11 +228,15 @@ function intervalHandler(timerInstance) {
 
 	if (timerInstance.config.countdown) {
 		timerInstance.totalSeconds = timerInstance.config.duration - timerInstance.totalSeconds;
-
-		if (timerInstance.totalSeconds === 0) {
+		
+		//Added isCallbackCalled flag, to avoid multiple calls to the callback.
+		if (!timerInstance.isCallbackCalled && timerInstance.totalSeconds === 0) {
```

Comment:
Do we really need the check for `isCallbackCalled`? We are clearing the interval on the next line. Once the interval is cleared the `callback` should not be called. Here s an example usage of the countdown with duration https://jsbin.com/noyijuqice/edit?js,console,output Lemme know if I m missing something.

Reply:
Removed the flag.